### PR TITLE
fix docs typo

### PR DESCRIPTION
**Pull request by CommanderStorm**

### CODE REVIEW

**Review by nyurik — APPROVED** [review]
thx

File: docs/src/martin-cp.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Generating Tiles in Bulk
 
 `martin-cp` is a tool for generating tiles in bulk, from any source(s) supported by Martin, and save retrieved tiles
-into a new or an existing MBTiles file. `martin-cp` can be used to generate tiles for a large area or multiple areas (
-bounding boxes). If multiple areas overlap, it will ensure each tile is generated only once. `martin-cp` supports the
+into a new or an existing MBTiles file. `martin-cp` can be used to generate tiles for a large area or multiple areas
```
Comment (CommanderStorm) [inline docs/src/martin-cp.md]:
```suggestion
into a new or an existing MBTiles file. `martin-cp` can be used to generate tiles for a large area or multiple areas
```

**Comment by nyurik** [comment]
CI currently excludes markdown-only tests to save on the runners. We probably should have some bypass in the CI so it thinks everything passed

**Comment by nyurik** [comment]
P.S. Please submit PRs only from your own fork - this is a requirement for all contributors including the core ones.  See https://gist.github.com/nyurik/4e299ad832fd2dd43d2b27191ed3ec30

**Comment by CommanderStorm** [comment]
Okay, will do for the future.

Btw: you/other maplibre Governing Board members can enforce this via a branch protection rule:

image

We should likely create a contributing guide which includes above, will steal the wording and come up with a first draft ^^

**Pull request merged**